### PR TITLE
#96 Add logo and summary  

### LINKED ISSUE #96: Add a logo and a project summary
This project needs to have a logo and a catchy project summary at the start of the README! The logo should be designed in SVG format. Here are some resources:

http://logodust.com/

http://openlogos.org/

Please note that the first site has logos that we can download and try to incorporate. The second page only has logos that we can use for inspiration as they have already been claimed. Once we have a great logo, I would like to start making stickers that we can share with developers!

### PR DESCRIPTION
**Pull request by dubswalker**
This logo was created using the Adobe Suite's Fireworks, Photoshop and exported with Illustrator. After briefly meeting with Professor Kapfhammer the decision was to create a logo in a "flat icon" style. The drawn sketch of the logo included an alligator with a graduation cap. This is to indicate that this logo can be used for college undergrads, grads and professionals.

@baldeosinghm will add summary

Fixes #96.

### CODE REVIEW

**Comment by dubswalker** [comment]
> Nice logo!

Thanks I hope it's not too bad. I tried to make it in a way that indicated it can be used for college undergrads, grads and professionals. I had another iteration, but it did not look as professional. Summary will be added soon.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+<p align="center">
+  <img width="820" height="500" src="https://github.com/dubswalker/gatorgrader/blob/master/gatorgraderlogo.svg">
```
Comment (Michionlion) [inline README.md]: This needs to reflect the url of the final image -- where will it be once it is merged to master?

Additionally, you should use markdown, not inline html here, unless there is some special reason.

Comment (Michionlion) [reply README.md]: @dubswalker Has there been any progress on this? Before it can be merged to master this must be fixed.

Comment (dubswalker) [reply README.md]: Changed it to reflect final url and got rid of incline html. 

Sorry for the delay. If it needs to be changed again let me know! I will be very responsive to this now that my computer works again.

**Pull request merged**